src/ast/nodes/CallExpression.ts

```diff
@@ -33,6 +33,9 @@ export default class CallExpression
 	/** Marked with #__PURE__ annotation */
 	declare annotationPure?: boolean;
 
+	private _evalWarned = false;
+	private _namespaceWarned = false;
```

Comment:
Following my other suggestion below, this could be

```suggestion
	private get hasCheckedForWarnings(): boolean {
		return isFlagSet(this.flags, Flag.checkedForWarnings);
	}

	private set hasCheckedForWarnings(value: boolean) {
		this.flags = setFlag(this.flags, Flag.checkedForWarnings, value);
	}
```

Then you just need to extend `BitFlags.ts` accordingly.